sotodlib/tod_ops/demodulation.py

```diff
@@ -0,0 +1,264 @@
+import numpy as np
```

Comment:
Could you add a comment string up here about what this module is for. 

"""
Functions for demodulating general sinusoidal modulation in timestreams. Often used in lab testing or with optical choppers. 
"""

I don't want this confused with any HWP demodulators because those will be different.

Reply:
done